secure_message/services/party_service.py

```diff
@@ -41,18 +55,16 @@ def get_user_details(uuid):
                           "telephone": "",
                           "status": "",
                           "sampleUnitType": "BI"}
+            logger.info("UUID is BRES")
             return party_dict, 200
 
-        url = current_app.config['RAS_PARTY_GET_BY_RESPONDENT'].format(current_app.config['RAS_PARTY_SERVICE'], uuid)
-        party_data = requests.get(url,
-                                  auth=current_app.config['BASIC_AUTH'],
-                                  verify=False)
-
-        logger.debug('Party get user details result',
-                     status_code=party_data.status_code,
-                     reason=party_data.reason,
-                     text=party_data.text,
-                     url=url)
+        if uuid not in self.users_cache:
+            logger.info("Executing call to the  party service ")
```

Comment:
Might be worth adding the `uuid` to the log here.

Reply:
+1 Definitely need contexts in logging